man/rgbasm.5

```diff
@@ -412,12 +412,21 @@ There are a number of escape sequences you can use within a string:
 .Pp
 Multi-line strings are contained in triple quotes
 .Pq Ql \&"\&"\&"for instance\&"\&"\&" .
-Escape sequences work the same way in multi-line strings; however, literal newline
-characters will be included as-is, without needing to escape them with
+Escape sequences work the same way in multi-line strings; however, literal newline characters will be included as-is, without needing to escape them with
 .Ql \er
 or
 .Ql \en .
 .Pp
+Raw strings are prefixed by a hash
+.Sq # .
+Inside them, backslashes and braces are treated like regular characters, so they will not be expanded as macro arguments, interpolated symbols, or escape sequences.
+For example, the raw string
+.Ql #"\t\1{s}\"
+is equivalent to the regular string
+.Ql "\\t\\1\{s}\\" .
+(Note that this prevents raw strings from including the double quote character.)
```

Comment:
The solution I've seen Rust and I *think* Python adopt for this, is that the closing quote must be followed by as many `#`s that preceded the opening one. Should we go with that?

Reply:
I shy away from the idea of putting `#`-counting code in the lexer. As it is, triple-quoted `#"""raw "string"s"""` can contain double quotes, just not three in a row. In the unlikely event that you need triple quotes in a raw string, just use `STRCAT`.

Reply:
Codegen backends may hate us for this, but ok let's go with that.